src/main/java/com/onepagecrm/models/EmailMessage.java

```diff
@@ -0,0 +1,127 @@
+package com.onepagecrm.models;
+
+import com.onepagecrm.exceptions.OnePageException;
+import com.onepagecrm.models.internal.DeleteResult;
+import com.onepagecrm.models.serializers.DeleteResultSerializer;
+import com.onepagecrm.net.ApiResource;
+import com.onepagecrm.net.Response;
+import com.onepagecrm.net.request.DeleteRequest;
+import com.onepagecrm.net.request.Request;
+import java.io.Serializable;
+
+
+
+/**
+ * Created by Anton S. on 05/07/2018
+ */
+
+public class EmailMessage extends ApiResource implements Serializable {
+
+    private String id;
+    private String contactId;
+    private String sendTime;
+    private String messageId;
+    private String sender;
+
+    private EmailRecipients recipients;
+    private String subject;
+    private String plainContent;
+    private String status;
+
+    public DeleteResult delete(String contactId) throws OnePageException {
+        // todo fix endpoint
+        Request request = new DeleteRequest(withId(CONTACT_EMAILS_ENDPOINT.replace("{id}", contactId)));
```

Comment:
@fraggjkee - endpoint looks like it should be generated okay to me!
```
DELETE .../api/v3/contacts/{contact_id}/email_messages/{email_id}.json
```
Right?

Reply:
Yep, wasn't there yet so I got 404.